pages/[network]/profile/payments.tsx

```diff
@@ -68,16 +68,24 @@ export default function Payments() {
           <span className="family-Regular h4 text-white">{t("main-nav.nav-avatar.payments")}</span>
 
           <FlexRow className="align-items-center">
-            <span className="caption-medium text-white mr-2">{t("labels.recivedintotal")}</span>
-            <div className="caption-large bg-dark-gray py-2 px-3 border-radius-8">
-              <span className="text-white">
-              {`${hasNoConvertedToken && "~" || ""} ${formatNumberToCurrency(totalEuro)}`}
-              </span>
-
-              <span className="text-gray ml-1">
-                {t("currencies.euro")}
+          {!hasNoConvertedToken ? (
```

Comment:
_don't use negatives on conditionals as it complicates everything when the variable name is also a negative, since it creates a double-negative when reading, probably inducing the reader in error_

```ts
setIsConverted(true|false)
{ isConverted ? "show things" : "show warning" }
```